docs/source/user_guide/fortran_c.rst

```diff
@@ -0,0 +1,78 @@
+##############
+Model Inteface
+##############
+
+Fortran C Interface Example
+---------------------------
```

Comment:
I don't see where this ends up in the documentation.

Reply:
The section is a place holder for now ...

Reply:
ah, that makes sense. thanks!